CMake/ITKSphinxExamplesExternalData.cmake

```diff
@@ -20,17 +21,59 @@ list(APPEND ExternalData_OBJECT_STORES
   "${CMAKE_SOURCE_DIR}/.ExternalData"
   )
 
+set(ITK_SPHINX_EXAMPLES_DATA_RELEASED_ROOT_CID bafybeicjaaz3xi2qso3kdirmy5ixoxegvfp3bm7o44ytoh6egj5m6wgeuu)
+set(ExternalData_URL_ALGO_CID_lower cid)
 set(ExternalData_URL_TEMPLATES "" CACHE STRING
   "Additional URL templates for the ExternalData CMake script to look for testing data. E.g.
 file:///var/bigharddrive/%(algo)/%(hash)")
 mark_as_advanced(ExternalData_URL_TEMPLATES)
-list(APPEND ExternalData_URL_TEMPLATES
-  # Data published on GitHub Pages
-  "https://insightsoftwareconsortium.github.io/ITKTestingData/%(algo)/%(hash)"
+if(NOT ITK_FORBID_DOWNLOADS)
+  list(APPEND ExternalData_URL_TEMPLATES
+    # Local IPFS gateway
+    "http://127.0.0.1:8080/ipfs/%(hash)"
 
-  # Data published on Girder
-  "https://data.kitware.com:443/api/v1/file/hashsum/%(algo)/%(hash)/download"
-  )
+    # Released data rsync'd to Kitware's Apache web server
+    "https://itk.org/files/ExternalData/%(algo)/%(hash)"
+
+    # Restricted gateway with released data
+    "https://itk.mypinata.cloud/ipfs/${ITK_SPHINX_EXAMPLES_DATA_RELEASED_ROOT_CID}/Objects/CID/%(hash)"
+
+    # New data on web3.storage
+    "https://w3s.link/ipfs/%(hash)"
+
+    # Released data on web3.storage
+    "https://${ITK_SPHINX_EXAMPLES_DATA_RELEASED_ROOT_CID}.ipfs.w3s.link/Objects/CID/%(hash)"
+
+    # Released data on estuary.tech
+    "https://${ITK_SPHINX_EXAMPLES_DATA_RELEASED_ROOT_CID}.ipfs.dweb.link/Objects/CID/%(hash)"
+
+    # Gateway for arbitrary new files, uploaded to web3.storage
+    "https://dweb.link/ipfs/%(hash)"
+
+    # Protocol Labs gateway
+    "https://ipfs.io/ipfs/%(hash)"
+
+    # Cloudflare gateway
+    "https://cloudflare-ipfs.com/ipfs/%(hash)"
+
+    # DataLad repository on gin.g-node.org
+    "https://gin.g-node.org/InsightSoftwareConsortium/ITKSphinxExamplesData/raw/main/Objects/CID/%(hash)"
+    )
+endif()
 
 # Tell ExternalData commands to transform raw files to content links.
-set(ExternalData_LINK_CONTENT SHA512)
+# TODO: Condition this feature on presence of our pre-commit hook.
```

Comment:
TODO to be addressed in this PR or added to backlog?

Reply:
This is not necessary -- I will remove it.